src/ResourceManager/RedisCache/Commands.RedisCache/Models/RedisCacheClient.cs

```diff
@@ -41,11 +41,11 @@ public RedisCacheClient(AzureContext context)
         }
         public RedisCacheClient() { }
 
-        public RedisResourceWithAccessKey CreateOrUpdateCache(string resourceGroupName, string cacheName, string location, string skuFamily, int skuCapacity, string skuName,
+        public RedisResource CreateCache(string resourceGroupName, string cacheName, string location, string skuFamily, int skuCapacity, string skuName,
```

Comment:
This is used by New cmdlet for calling into C# SDK. There is no reason why powershell customer would call in this directly